docs/scripts/translate-docs.py

```
@@ -0,0 +1,623 @@
+import json
+import argparse
+import hashlib
+import re
+import time
+from pathlib import Path
+
+try:
+    import requests
+except ImportError:
+    print("Installing requests...")
+    os.system("pip3 install requests -q")
```

Ruff (0.15.12)

[error] Starting a process with a shell: seems safe, but may be changed in the future; consider rewriting without `shell`

(S605)